partials/cli-plugins/_insight-ingress-configuration.hbs.md

```diff
@@ -0,0 +1,32 @@
+<!-- Configure certificate and end point for insight cli when ingress is enabled -->
+
+To get a certificate, run:
```

Comment:
`To get the certificate`

Because there is only one right certificate to fetch.